internal/splunk/common.go

```diff
@@ -66,7 +66,7 @@ type Event struct {
 
 // IsMetric returns true if the Splunk event is a metric.
 func (e *Event) IsMetric() bool {
-	return e.Event == HecEventMetricType || (e.Event == nil && len(e.GetMetricValues()) > 0)
+	return e.Event == HecEventMetricType || ((e.Event == nil || e.Event == "") && len(e.GetMetricValues()) > 0)
```

Comment:
Can you test against splunk if this is the correct check? I was told: `As tested out, the event field is optional. When presented, it can also be any string and does not have to be of value metric. A possible way of updating this would be to change`